### PR TITLE
Add One Patient as Example from the Generator

### PR DESCRIPTION
**Pull request by alexanderkiel**
Also download the IPS IG pack in order to have access to the value set
http://hl7.org/fhir/uv/ips/ValueSet/current-smoking-status-uv-ips.

### CODE REVIEW

File: examples/transaction-0.json [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+            }
+          ]
+        }
+      }
+    }
+  ],
+  "id": "6c4f4112-5f04-4ce0-a843-540698e891dd",
+  "resourceType": "Bundle",
+  "type": "transaction"
+}
```
Comment (noemide) [inline examples/transaction-0.json]: Missing newline

File: .travis.yml [hunk 2]
```diff
@@ -1,7 +1,8 @@
 language: java
 before_script:
 - wget https://fhir.github.io/latest-ig-publisher/org.hl7.fhir.validator.jar -O validator.jar
+- wget http://hl7.org/fhir/uv/ips/2019Sep/validator.pack -O ips.pack
```
Comment (noemide) [inline .travis.yml]: Why do we need the IPS?

**Review by noemide — CHANGES_REQUESTED** [review]

File: examples/transaction-0.json (hunk 1 above)
Comment (alexanderkiel) [reply examples/transaction-0.json]: Ah shit. You are right. :-) Have to add it to the generator.

**Review by noemide — DISMISSED** [review]

File: examples/transaction-0.json (hunk 1 above)
Comment (alexanderkiel) [reply examples/transaction-0.json]: Fixed

**Review by noemide — APPROVED** [review]

**Pull request merged**